apps/remix-ide/src/app/files/fileManager.js

```diff
@@ -563,8 +563,6 @@ class FileManager extends Plugin {
     if (file.startsWith('browser')) {
       return this._deps.filesProviders.browser
     }
-    const provider = this._deps.filesProviders.workspace
```

Comment:
do you need to remove this?

Reply:
yes we don't need to show that error anymore